### PR TITLE
[Azure App Configuration] Update swagger to include LRO header for snapshot create.

### PR DESCRIPTION
**Pull request by jimmyca15**
Update swagger to include LRO header for snapshot create.

### CODE REVIEW

**Review by JeffreyRichter — APPROVED** [review]

**Comment by jimmyca15** [comment]
@jhendrixMSFT Can you merge this? Not sure if I need to sign-off in some way.

**Pull request merged**
